pyro/distributions/torch.py

```diff
@@ -101,13 +100,31 @@ def __init__(self, a, b):
     if _Dist is torch.distributions.Distribution:
         continue
 
+    try:
+        _PyroDist = locals()[_name]
+    except KeyError:
+
+        class _PyroDist(_Dist, TorchDistributionMixin):
+            pass
+
+        _PyroDist.__name__ = _name
+        locals()[_name] = _PyroDist
+
+    _PyroDist.__doc__ = '''
+    Wraps :class:`torch.distributions.{}` with
+    :class:`~pyro.distributions.torch_distribution.TorchDistributionMixin`.
+    '''.format(_Dist.__name__)
+
     __all__.append(_name)
-    if _name in dir():
-        continue
 
-    class _PyroDist(_Dist, TorchDistributionMixin):
-        pass
 
-    _PyroDist.__name__ = _name
-    locals()[_name] = _PyroDist
-    del _PyroDist, _name, _Dist
+# Create sphinx documentation.
+__doc__ = '\n\n'.join([
```

Comment:
This was the only way I was able to get these distributions to show up in our sphinx toc.

Reply:
oof. gross